bench-tx/src/utils.rs

```diff
@@ -83,7 +82,7 @@ pub fn get_account_with_default_account_code(
     assets: Option<Asset>,
 ) -> Account {
     let account_code_src = DEFAULT_ACCOUNT_CODE;
-    let assembler = TransactionKernel::assembler();
+    let assembler = TransactionKernel::assembler().with_debug_mode(true);
```

Comment:
Do we need the `.with_debug_mode()` here?

Reply:
I was trying to use `debug.stack` decorator to get the stack state in this benchmarks, but it doesn't work anyway (btw I'm still not sure why), so we can remove it.